src/ip-fragment.c

```diff
@@ -92,6 +97,51 @@ static int do_fragmentation(struct pg_ip_fragment_state *state,
 	return ret;
 }
 
+static inline int do_reasemble(struct pg_ip_fragment_state *state,
+			       struct rte_mbuf **pkts, struct pg_brick_side *s,
+			       enum pg_side from, uint64_t *pkts_mask,
+			       struct pg_error **errp)
+{
+	int j = 0;
+	uint64_t snd_mask = 0;
+	uint64_t remove_mask = 0;
+	int ret = 0;
+
+	PG_FOREACH_BIT(*pkts_mask, i) {
+		struct  eth_ipv4_hdr *pkt_buf =
+			rte_pktmbuf_mtod(pkts[i],
+					 struct eth_ipv4_hdr *);
+
+		if (unlikely(rte_ipv4_frag_pkt_is_fragmented(&pkt_buf->ip))) {
+			struct rte_mbuf *tmp;
+
+			remove_mask |= (ONE64 << i);
+			tmp = rte_ipv4_frag_reassemble_packet(state->tbl,
+							      &state->dr,
+							      pkts[i],
+							      rte_rdtsc(),
+							      &pkt_buf->
+							      ip);
+			if (tmp) {
+				state->pkts_out[j] = tmp;
+				snd_mask |= (ONE64 << j);
+				++j;
+			}
+		}
+	}
+
+	if (unlikely(snd_mask)) {
```

Comment:
same here: It seems hard to evaluate if receiving fragmented packets is "unlikely"